Mathlib/Logic/Equiv/Set.lean

```diff
@@ -174,6 +174,25 @@ def image {α β : Type*} (e : α ≃ β) (s : Set α) :
   left_inv x := by simp
   right_inv y := by simp
 
+section order
+
+variable {α β : Type*} [Preorder α] [Preorder β] {e : α ≃ β} (s : Set α)
+
+lemma monotone_image (hs : Monotone e) :
+    Monotone (e.image s) :=
+  hs.comp (Subtype.mono_coe _)
+lemma antitone_image (hs : Antitone e) :
+    Antitone (e.image s) :=
+  hs.comp_monotone (Subtype.mono_coe _)
+lemma strictMono_image (hs : StrictMono e) :
+    StrictMono (e.image s) :=
+  hs.comp (Subtype.strictMono_coe _)
+lemma strictAnti_image (hs : StrictAnti e) :
+    StrictAnti (e.image s) :=
+  hs.comp_strictMono (Subtype.strictMono_coe _)
```

Comment:
I think the naming convention says these should all be named in reverse.

Also can you format as

```
lemma monotone_image (hs : Monotone e) : Monotone (e.image s) :=
  hs.comp (Subtype.mono_coe _)
```
to save some space?